tests/functional/test_install_wheel.py

```diff
@@ -129,6 +129,42 @@ def test_install_wheel_with_target(script, data):
     )
 
 
+@pytest.mark.network
+def test_install_wheel_with_target_and_data_files(script, data):
+    """
+    Test for issue #4092. It will be checked that a data_files specification in
+    setup.py is handled correctly when a wheel is installed with the --target
+    option.
+
+    The setup() for the wheel 'pip_issue_4092-1.0-py2.py3-none-any.whl' is as
+    follows ::
+
+        setup(
+            name='pip_issue_4092',
+            version='1.0',
+            packages=['pip_issue_4092'],
+            data_files=[
+                (r'packages1', ['pip_issue_4092/README.txt']),
+                (r'packages2', ['pip_issue_4092/README.txt'])
+            ]
+        )
+    """
+    script.pip('install', 'wheel')
```

Comment:
If you directly provide the wheel via `data.packages`, you don't need to have `wheel` installed so you can drop this (and `@pytest.mark.network`).